runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/artifact/AbstractArtifactStagingService.java

```diff
@@ -77,25 +79,27 @@ public abstract String getArtifactUri(String stagingSessionToken, String encoded
   public void commitManifest(
       CommitManifestRequest request, StreamObserver<CommitManifestResponse> responseObserver) {
     try {
-      String stagingSessionToken = request.getStagingSessionToken();
-      ProxyManifest.Builder proxyManifestBuilder =
-          ProxyManifest.newBuilder().setManifest(request.getManifest());
-      for (ArtifactMetadata artifactMetadata : request.getManifest().getArtifactList()) {
-        proxyManifestBuilder.addLocation(
-            Location.newBuilder()
-                .setName(artifactMetadata.getName())
-                .setUri(getArtifactUri(stagingSessionToken, encodedFileName(artifactMetadata)))
-                .build());
-      }
-      try (WritableByteChannel manifestWritableByteChannel = openManifest(stagingSessionToken)) {
-        manifestWritableByteChannel.write(
-            CHARSET.encode(JsonFormat.printer().print(proxyManifestBuilder.build())));
+      String retrievalToken = NO_ARTIFACTS_STAGED_TOKEN;
```

Comment:
Style: I would prefer

```
if (request.getManifest().getArtifactCount() == 0) {
  retrievalToken = NO_ARTIFACTS_STAGED_TOKEN
} else {
  ...
}

to avoid re-assignment. 
```